mangaki/mangaki/management/commands/compare.py

```diff
@@ -105,6 +106,6 @@ class Command(BaseCommand):
     help = 'Compare recommendation algorithms'
 
     def handle(self, *args, **options):
-        experiment = Experiment(1046)  # 1706
-        experiment.display_ranking()
+        experiment = Experiment()  # 1706
+        #experiment.display_ranking()
```

Comment:
It prints a lot (50 000) of predictions. It's not very convenient to examine the results of algorithms which are above, so I comment it.